eodatasets3/prepare/landsat_l1_prepare.py

```diff
@@ -362,7 +430,7 @@ def prepare_and_write(
         for section, fields in _COPYABLE_MTL_FIELDS[collection_key]:
             for field in fields:
                 value = mtl_doc[section].get(field)
-                if value is not None:
+                if value is not None and p.properties.get(f"landsat:{field}") is None:
```

Comment:
This was due to multiple sections having the same property, but with different values because the level2 value, which is given precedence in the new logic if it exists, was then being overwritten by a level1 value. My thinking was that if the precedence logic is correct then the first value for something should be accurate.